unitTests/test_basicio.cpp

```diff
@@ -0,0 +1,24 @@
+#include <exiv2/basicio.hpp>
+
+#include <cmath>
```

Comment:
[minor] These headers seem to not be needed, right?. The only usage from standard functions that I see is `memset` which would be available in `#include <cstring>`